src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/InputHelper.java

```diff
@@ -58,20 +59,25 @@ public static SampleCase sampleCase(List<Item<ScannedDocument>> scannedDocuments
             "date-of-birth",
             "contact-number",
             "email",
-            new Address(
-                "addr-line-1",
-                "addr-line-2",
-                "addr-line-3",
-                "post-code",
-                "post-town",
-                "county",
-                "country"
-            ),
+            address(""),
             scannedDocuments,
             "er-id"
         );
     }
 
+    @NotNull
```

Comment:
Why do we need this annotation?

Reply:
It is auto-generated when method is extracted in IntelliJ. Removed now.